integration/simulation/simulation_in_mem_test.go

```diff
@@ -24,11 +36,78 @@ func TestInMemoryMonteCarloSimulation(t *testing.T) {
fffffff8111156106f9576106f8610c18565b5b60208301915083600182028301111561071557610714610c22565b5b9250929050565b600082601f83011261073157610730610c1d565b5b813561074184826020860161065a565b91505092915050565b60008135905061075981610c75565b92915050565b60006020828403121561077557610774610c31565b5b60006107838482850161069c565b91505092915050565b600080602083850312156107a3576107a2610c31565b5b600083013567ffffffffffffffff8111156107c1576107c0610c2c565b5b6107cd858286016106c6565b92509250509250929050565b6000602082840312156107ef576107ee610c31565b5b600082013567ffffffffffffffff81111561080d5761080c610c2c565b5b6108198482850161071c565b91505092915050565b6000806040838503121561083957610838610c31565b5b60006108478582860161074a565b9250506020610858858286016106b1565b9150509250929050565b6000806040838503121561087957610878610c31565b5b60006108878582860161074a565b92505060206108988582860161074a565b9150509250929050565b60006108ae838361092b565b905092915050565b60006108c182610a71565b6108cb8185610a94565b9350836020820285016108dd85610a61565b8060005b8581101561091957848403895281516108fa85826108a2565b945061090583610a87565b925060208a019950506001810190506108e1565b50829750879550505050505092915050565b600061093682610a7c565b6109408185610aa5565b9350610950818560208601610b24565b61095981610c36565b840191505092915050565b600061096f82610a7c565b6109798185610ab6565b9350610989818560208601610b24565b61099281610c36565b840191505092915050565b6109a681610b0b565b82525050565b600060208201905081810360008301526109c681846108b6565b905092915050565b600060208201905081810360008301526109e88184610964565b905092915050565b6000602082019050610a05600083018461099d565b92915050565b6000610a15610a26565b9050610a218282610b89565b919050565b6000604051905090565b600067ffffffffffffffff821115610a4b57610a4a610be9565b5b610a5482610c36565b9050602081019050919050565b6000819050602082019050919050565b600081519050919050565b600081519050919050565b6000602082019050919050565b600082825260208201905092915050565b600082825260208201905092915050565b600082825260208201905092915050565b6000610ad282610aeb565b9050919050565b6000610ae482610aeb565b9050919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b6000819050919050565b82818337600083830152505050565b60005b83811015610b42578082015181840152602081019050610b27565b83811115610b51576000848401525b50505050565b60006002820490506001821680610b6f57607f821691505b60208210811415610b8357610b82610bba565b5b50919050565b610b9282610c36565b810181811067ffffffffffffffff82111715610bb157610bb0610be9565b5b80604052505050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052602260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b600080fd5b600080fd5b600080fd5b600080fd5b600080fd5b600080fd5b6000601f19601f8301169050919050565b610c5081610ac7565b8114610c5b57600080fd5b50565b610c6781610ad9565b8114610c7257600080fd5b50565b610c7e81610b0b565b8114610c8957600080fd5b5056fea2646970667358221220382f61b400ade4111e9bd34b846340e65f0eee85ffc7ec3ea73d057688847afd64736f6c63430008070033`
```

Comment:
@otherview Can you pull all of these constants out? Can you also leave a comment explaining where these bytes come from? Is there any way we can generate the contract bytes instead of hardcoding them?

Reply:
Added a comment in the readme. Short answer: it's a bit of a bollocks to automate the bytecode compilation, so going to remix and manually copying it, is probably the easiest solution given the number of times we're changing the contract.

Reply:
Pulled the constats out, next iteration its refactorable to an auto-deploy script